engine/requirements.txt

```diff
@@ -416,6 +416,10 @@ rsa==4.9
     # via google-auth
 s3transfer==0.10.0
     # via boto3
+setuptools==72.1.0
+    # via
+    #   apscheduler
+    #   opentelemetry-instrumentation
```

Comment:
I guess there's not much we can do here? I assume this means that `apscheduler` and `opentelemetry-instrumentation` both have a dependency on `setuptools`?